documentation-site/examples/breadcrumbs/pseudo.tsx

```diff
@@ -1,13 +1,14 @@
 import * as React from 'react';
 import {Breadcrumbs} from 'baseui/breadcrumbs';
 import {StyledLink as Link} from 'baseui/link';
+import {StyleObject} from 'styletron-react';
 
 export default function Example() {
   return (
     <Breadcrumbs
       overrides={{
         ListItem: {
-          style: ({$itemIndex, $theme}) => {
+          style: ({$itemIndex, $theme}): StyleObject => {
```

Comment:
Was this required after the type def changes? Wondering if I should categorize this as a breaking api change

Reply:
double-checked it - this change is not actually required for current type definitions to work

let me revert it